.github/workflows/c3-e2e-dependabot.yml

```diff
@@ -55,7 +55,8 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
         with:
-          fetch-depth: 0
+          fetch-depth: 3
+          ref: ${{ github.head_ref }}
```

Comment:
I think the correct fix is to change the trigger to `pull_request` rather than `pull_request_target`, which according to the docs runs the job in the context of the base of the PR rather than the HEAD.

Reply:
sounds reasonable to me, thanks 🙂 

I've update the trigger, please have another look 🙏